### PR TITLE
GH: Run build pipeline only on relevant changes

### PR DESCRIPTION
**Pull request by pnahratow**

### CODE REVIEW

File: .github/workflows/build_release.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ on:
     branches:
       - main
   pull_request:
+    paths:
+      - 'lq1/**'
+      - 'qsrc/**'
+      - 'texture-wads/**'
```
Comment (MotoLegacy) [inline .github/workflows/build_release.yml:13]: @lavenderdotpet Do we use the scripts in the `bin/` path for any reason?

Comment (lavenderdotpet) [reply .github/workflows/build_release.yml:13]: i mainly have it there to keep track of how to do some thing in qpakman

Comment (MotoLegacy) [reply .github/workflows/build_release.yml:13]: Thanks! We should probably remove it then to reduce repository clutter, if you could keep it elsewhere that would be very much appreciated. Just wanted to make sure we didn't need to include it into this path.

**Review by MotoLegacy — APPROVED** [review]
Thanks for this, a much needed change!

**Review by lavenderdotpet — APPROVED** [review]
looks good to me

**Pull request merged**